srcjs/input_binding_date.js

```diff
@@ -196,6 +196,7 @@ $.extend(dateInputBinding, {
 
     // Workaround for same issue as in _setMin.
     if (typeof date.toDateString !== 'function') return;
+    if (curValue === null ) return;
```

Comment:
This'll defend against an `undefined` value as well

```suggestion
    if (!curValue) return;
```